src/Artsy/Router/Components/Boot.tsx

```diff
@@ -28,7 +29,16 @@ export interface BootProps {
   headTags?: JSX.Element[]
 }
 
-const { GlobalStyles } = injectGlobalStyles(`
+// Build global responsive styles
+const reactResponsiveStyles = createMediaStyle()
+
+// FIXME: When we update to latest @types/styled-components `suppressMultiMountWarning`
+// issue will be fixed
+const { GlobalStyles } = injectGlobalStyles<{
+  suppressMultiMountWarning: boolean
+}>(`
+  ${reactResponsiveStyles}
+
   h1 {
```

Comment:
should the `reactResponsiveStyles` come _after_ this h1 reset. Also, why is there just an h1 here? 🤔

Reply:
Ah you're right -- this was from a rebase. 

Re `h1`, this can eventually go in palette. This was a fix for @eessex from last week:
- https://github.com/artsy/reaction/pull/1951
- https://github.com/artsy/reaction/pull/1970